tests/unit/style/parsing/image.rs

```diff
@@ -10,6 +10,22 @@ use style::values::specified::image::*;
 use url::Url;
 
 #[test]
+fn test_linear_gradient() {
+    // Parsing from the right
+    assert_roundtrip_with_context!(Image::parse, "linear-gradient(to left, red, green)",
+                                                   "linear-gradient(4.712389rad, red, green)");
```

Comment:
Eeks, didn't notice this. Somewhat surprised that `mach test-tidy` didn't catch that. :thinking: